supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/PipelinesApiAsyncBackendJobExecutionActor.scala

```diff
@@ -86,4 +87,16 @@ class PipelinesApiAsyncBackendJobExecutionActor(standardParams: StandardAsyncExe
       })
     case _ => List.empty
   }
+
+  override def generateSingleFileOutputs(womFile: WomSingleFile, fileEvaluation: FileEvaluation) = {
```

Comment:
Do we have a test for the original problem that this change was intended to fix when it was made?

Reply:
conformance test 55 (on PAPI2) is the test